src/watchdog.js

```diff
@@ -44,13 +42,27 @@ export default class Watchdog {
 		this.crashes = [];
 
 		/**
-		 * Crash number limit (defaults to `3`). After this limit is reached the editor is not restarted by the watchdog.
-		 * This is to prevent an infinite crash loop.
+		 * Specifies the watchdog state.
```

Comment:
Actually, it's a state of the editor, I'm not sure how to document it.

Reply:
Agree. We should be documented as a state if the editor handled by the watchdog.

Reply:
Also, it looks like a good place to describe what each state means. The description you put in "Additional information" in this PR is very nice, I think it should go here.

Reply:
I've added it here and in the API docs.